x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/hooks/use_create_analytics_form/state.ts

```diff
@@ -302,7 +301,7 @@ export function getFormStateFromJobConfig(
       : analyticsJobConfig.source.index,
     modelMemoryLimit: analyticsJobConfig.model_memory_limit,
     maxNumThreads: analyticsJobConfig.max_num_threads,
-    includes: analyticsJobConfig.analyzed_fields.includes,
+    includes: analyticsJobConfig.analyzed_fields?.includes || [],
```

Comment:
I think there is still a problem when cloning a job created by the API if it has a non-default query. In this case on cloning, it switches to the advanced editor but seems to delete most of the cloned config.

For example, using the `mushroom` data set, I created this job in dev tools:

```
PUT _ml/data_frame/analytics/mushroom_edibility_query
{
  "description": "",
  "source": {
    "index": "mushroom",
    "query": {
      "bool": {
        "should": [
          {
            "match": {
              "bruises": "t"
            }
          }
        ],
        "minimum_should_match": 1
      }
    }
  },
  "dest": {
    "index": "mushroom_edibility_query",
    "results_field": "ml"
  },
  "analysis": {
    "classification": {
      "dependent_variable": "edibility",
      "num_top_feature_importance_values": 5,
      "training_percent": 14,
      "prediction_field_name": "edibility_prediction",
      "randomize_seed": -3740970564557279700,
      "num_top_classes": 2
    }
  },
  "model_memory_limit": "24mb"
}
```

but on cloning, the advanced editor opens with this:

![image](https://user-images.githubusercontent.com/7405507/94807192-349d8800-03e7-11eb-89d3-490452d4dad6.png)